lib/netutils.py

```diff
@@ -61,12 +60,6 @@
 _STRUCT_UCRED = "iII"
 _STRUCT_UCRED_SIZE = struct.calcsize(_STRUCT_UCRED)
 
-# Workaround a bug in some linux distributions that don't define SO_PEERCRED
```

Comment:
I can confirm that in Ubuntu 14.04, 16.04 and 18.04, socket(7) refers to `SO_PEERCRED` so I don't see a need to retain this workaround from that perspective.